gapic/templates/noxfile.py.j2

```diff
@@ -32,7 +32,10 @@ ALL_PYTHON = [
     "3.14",
 ]
 
-DEFAULT_PYTHON_VERSION = ALL_PYTHON[-2]
+DEFAULT_PYTHON_VERSION = ALL_PYTHON[-1]
+# TOOD(https://github.com/googleapis/gapic-generator-python/issues/2450):
+# Add tests for Python 3.15 alpha1
+# https://peps.python.org/pep-0790/
```

Comment:
Why do we need this TODO here? The nox tests are parametrized below, so by changing the variables with the Python versions above, we can control what gets tested. I'd remove this TODO.

Reply:
It's clearer now that we're being explicit about the Python version. Thanks.